starter-code/src/clue.js

```diff
@@ -1,94 +1,192 @@
+    last_name:    "Plum",
+    color:        "purple",
+    description:  "Billionare video game designer",
+    age:          22,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+    occupation:   "Designer"
+    };
+    
+    
+    var missScarlet ={
+    first_name:   "Kasandra",
+    last_name:    "Scarlet",
+    color:        "red",
+    description:  "She is an A-list movie star with a dark past",
+    age:          31,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+    occupation:   "Actor"
+    
+    };
+    
+    var mrsPeacock = {
+    first_name:   "Eleanor",
+    last_name:    "Peacock",
+    color:        "blue",
+    description:  "She is from a wealthy family and uses her status and money to earn popularity",
+    age:          36,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation:   "Socialité"
+    };
+    
+    
+    var mrMustard = {
+    first_name:   "Jack",
+    last_name:    "Mustard",
+    color:        "yellow",
+    description:  "He is a former football player who tries to get by on his former glory",
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player"
+    };
+    
+    
+    //Weapons
+    
+    var weapon1=  {name: "rope", weight: 10};
+    var weapon2 = {name: "knife", weight: 8};
+    var weapon3 = {name: "candlestick", weight: 2};
+    var weapon4 = {name: "dumbbell", weight: 30};
+    var weapon5 = {name: "poison",weight: 2};
+    var weapon6 = {name: "bat", weight: 13};
+    var weapon7 = {name: "axe", weight: 15};
+    var weapon8 = {name: "trophy", weight: 25};
+    var weapon9 = {name: "pistol", weight: 20};
+
+
+
+    // Rooms
+var room1 = {name : "Dinning Room"};
+var room2 = {name : "Conservatory"};
+var room3 = {name : "Kitchen"};
+var room4 = {name : "Study"};
+var room5 = {name : "Library"};
+var room6 = {name : "Billiard Room"};
+var room7 = {name : "Lounge"};
+var room8 = {name : "Ballroom"};
+var room9 = {name : "Hall"};
+var room10 = {name: "Spa"};
+var room11 = {name: "Living Room"};
+var room12 = {name: "Observatory"};
+var room13 = {name: "Theater"};
+var room14 = {name: "Guest House"};
+var name15 = {name: "Patio"};
+
 
 // Characters Collection
 var charactersArray = [];
 
+charactersArray.push(mrGreen);
+charactersArray.push(drOrchid);
+charactersArray.push(profPlum);
+charactersArray.push(missScarlet);
+charactersArray.push(mrsPeacock);
+charactersArray.push(mrMustard);
+
+// console.log(charactersArray);
 // Rooms' Collection
 var roomsArray = [];
 
+roomsArray.push(weapon1);
+roomsArray.push(weapon2);
+roomsArray.push(weapon3);
+roomsArray.push(weapon4);
+roomsArray.push(weapon5);
+roomsArray.push(weapon6);
+roomsArray.push(weapon7);
+roomsArray.push(weapon8);
+
+//console.log(roomsArray);
 // Weapons Collection
 var weaponsArray = [];
+
+weaponsArray.push(room1);
```

Comment:
instead of writing the same method like this, just use this simple line and it will do the same : 

```js

weaponsArray.push(room1,room2,room3,room4, ect..) 

```